### PR TITLE
Minor fixes to make wavelet work on 32 bit arm

### PR DESCRIPTION
**Pull request by hasyimibhar**
- Fix compile errors due to math constants implicitly converted to int: https://github.com/golang/go/issues/24523#issuecomment-375973171
- Fix 64 bit atomic operations not working due to alignment issues: https://go101.org/article/memory-layout.html

### CODE REVIEW

**Comment by hasyimibhar** [comment]
This PR should fix #221.

File: wctl/counter.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import (
+	"sync"
+	"sync/atomic"
+	"unsafe"
+)
+
+type Counter struct {
+	n    uint64
+	lock sync.RWMutex
```
Comment (iwasaki-kenta) [inline wctl/counter.go]: I feel this is a good time for us to introduce https://github.com/uber-go/atomic. Thoughts?

Comment (hasyimibhar) [reply wctl/counter.go]: Good idea.

**Comment by hasyimibhar** [comment]
Dont merge it yet. Seems like theres still an issue when using uber's atomic package.

**Comment by hasyimibhar** [comment]
Okay it works now.

**Pull request merged**